### DISCUSSION TITLE
remix-pwa/sw should be in dependencies?

### DISCUSSION THREAD

**Discussion by dvnrsn** [ShafSpecs/remix-pwa#161] [Q&A] [answered]
When deploying with fly, I noticed a problem after `fly deploy` 

```
cannot find package '@remix-pwa/sw'
```

After moving the package to dependencies (from devDeps) it's working

**Comment by ShafSpecs** [comment, marked as answer]
Yeah, libraries required in your application like `sw`, `sync` and `push` should be installed as dependencies. The other one used in the build process like runtime and dev are to be installed as dev dependencies

**Reply by dvnrsn** [reply]
Hmm, yeah I must have missed something because I thought I followed something in the docs putting it dev. Ignore me.
